Add closedsource media driver to load according to env config

Yanfeng-Mi opened the pull request:
env var to control: ro.vendor.media.closesource

sysopenci commented:
Improper Commit Message
Tracked on not found in commit message,
    make sure Tracked-On: Jira-ticket  is present.

Yanfeng-Mi committed:
Add closedsource media driver to load according to android boot config

kernel cmdline to open: ro.boot.mcd (androidboot.mcd=true)

Tracked-On: OAM-130955
Signed-off-by: Mi, Yanfeng <yanfeng.mi@intel.com>

sysopenci commented:
Improper Commit Message
Tracked on not found in commit message,
    make sure Tracked-On: Jira-ticket  is present.

feijiang1 reviewed (approved):
LGTM

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
FAILURE: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /cactus-absp-or-local/celadon_umr0_master-merge/450
